docs/developer-docs/devquests.md

```diff
@@ -315,9 +317,9 @@ You must have developed sets for 10 or more supported platforms before opt-in. O
 
 1. That means both sets must be DIFFERENT SPORTS. You cannot do two baseball games, or two soccer games.
 2. Must be full sets containing all content and challenges appropriate to the game.
-3. No Homebrew/Hacks/Unlicensed
-4. New sets only: No Multi Set or Revisions
-5. Other picks are subject to DQ team approval if eligibility is questionable
+3. No Homebrew/Hacks/Unlicensed.
+4. New sets only: No Multi Set or Revisions.
```

Comment:
```suggestion
4. New sets only. No multiplayer subsets or revisions.
```
Let's go ahead and change this while we're here. I think we'll wind up rebranding "Multi" to "Multiplayer".